src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleView.cs

```diff
@@ -525,26 +525,6 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
             RemoveAndRecycleScrapInt();
         }
 
-        /// <summary>
-        /// you can override it to do something for style change.
-        /// </summary>
-        /// <since_tizen> 6 </since_tizen>
-        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public override void OnStyleChange(NUI.StyleManager styleManager, StyleChangeType change)
```

Comment:
@rabbitfor 
I've deleted OnStyleChanged because CustomView.OnStyleChange() is deprecated.
This is the reason for build warning[CS0627] https://docs.microsoft.com/ko-kr/dotnet/csharp/misc/cs0672.

CustomView.OnstyleChange->
https://github.com/Samsung/TizenFX/blob/4c6d61473e4a0fadde471ab1e6291810df4ec23a/src/Tizen.NUI/src/public/BaseComponents/CustomView.cs#L408-L409

Can I delete this code?

Reply:
It seems OK.